stockflux-securities-master/src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./Button.css";
+import PropTypes from "prop-types";
+
+const Button = ({ text, size, className, ...props }) => {
+  return (
+    <div className={`button-container ${size} ${className}`}>
+      <button {...props}>{text}</button>
+    </div>
+  );
+};
+
+Button.propTypes = {
+  size: PropTypes.oneOf(["extraSmall", "small", "large"]).isRequired,
```

Comment:
See above about keeping naming conventions consistent. If you want to accept `extraSmall` as a prop here, then you should at least pass `extra-small` to the `className` prop for consistency.

Reply:
Yep changed the name here now